README.md

```diff
@@ -103,6 +103,7 @@ Valid values are:
 - single
 - subdomain
 - subdirectory
+- skip
```

Comment:
I think `none` would be a better value for "this is no WP install".

Reply:
+1 for `none` as the config value.